articles/migrations/index.md

```diff
@@ -25,6 +25,85 @@ If you need help with the migration, create a ticket in our [Support Center](${e
 
 Current migrations are listed below, newest first. For migrations that have already been enabled see [Past Migrations](#past-migrations).
 
+### Introducing Lock v11, Auth0.js v9
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-04-01 |
+
+We’re continually improving the security of our service. As part of this, we are deprecating a set of APIs (`usernamepassword/login`, `/ssodata`, `/tokeninfo`, `/delegation`) used by Lock.js v8, v9, and v10 and and auth0.js, v6, v7, and v8. You should update your applications by **April 1, 2018**.
+
+We are offering two options for this update:
+
+1. We are moving towards a centralized login experience and are recommending this option moving forward. There are many benefits of using a centralized login experience, read more about how Centralized Login works [here](/hosted-pages/login). Moving to the centralized login experience is straightforward and will simplify your code base. Read [this guide](/guides/login/centralized-vs-embedded) to learn more about the benefits of centralized login vs embedded login.
```

Comment:
We shouldn't use `here` or `click here` for links (read more [here](https://www.w3.org/QA/Tips/noClickHere) 😆  ). It's not informative, it's bad for SEO and bad for accessibility.

Reply:
Also, I'm not a big fan of this kind of self-promotion: `Moving to the centralized login experience is straightforward and will simplify your code base.` We seem to have it all over the place for HLP, and it's generally bad practice to tell a user how simple something is all the time.

Reply:
`This guide` is a bad name for a link as well. Why not sth like this? `Learn more about the benefits of [centralized login vs embedded login](/guides/login/centralized-vs-embedded).`

Reply:
I agree that we should back off of the "it's simple" phrasing that we have in quite a few places, especially now that we have COA and custom domains as an alternative. It's enough to say that we recommend it because it's more secure.

In this specific case though, the statement is more of a statement of fact:

>will simplify your code base

By redirecting the user to HLP to do authentication, your code base is less complex as it doesn't include any auth stuff. It's not saying implementation is simpler, it's saying that less code is required in your application.